README.md

```diff
@@ -1 +1,39 @@
-# webpack
\ No newline at end of file
+# Leaderboard
+A project to display a simple score game, using API, You push the player name and his score, and upon clicking on refresh you will get all the results that are pushed to the API. 
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+[Live Demo Link](https://leaderboardgame.netlify.app/)
+
```

Comment:
- [ ] As the previous reviewer suggest kindly add a Get Started section with clear instructions on how to use the app